cmd/ejson/main.go

```diff
@@ -16,6 +17,8 @@ func execManpage(sec, page string) {
 }
 
 func main() {
+	// Encryption is expensive. We'd rather burn cycles on many cores than wait.
+	runtime.GOMAXPROCS(runtime.NumCPU())
```

Comment:
Meh, I want ejson to have as few options as possible. I want to pick one way and stick to it.